packages/lesspass-pure/src/store/getters.js

```diff
@@ -12,11 +12,10 @@ export const passwordURL = state => {
 };
 
 export const isDefaultProfile = state => {
-  let defaultProfile = true;
   for (let key in defaultOptions) {
-    if (defaultOptions[key] !== state.password[key]) {
-      defaultProfile = false;
+    if (state.defaultPassword[key] !== state.password[key]) {
+      return false;
     }
   }
-  return defaultProfile;
+  return true;
```

Comment:
I think you want `let key in state.defaultPassword` no?

Reply:
That's what I did in the first time, but it's not consistent with the original behaviour, as the state.defaultPassword object also includes `login` and `site`.
Concretely, it would make the password options pane appear as soon as we update the default login in the options (`PasswordGenerator`, l. 104), which I guess is not the desired behaviour.